sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -839,7 +839,7 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
       case logical.MapInPandas(func, output, child, isBarrier) =>
         execution.python.MapInPandasExec(func, output, planLater(child), isBarrier) :: Nil
       case logical.PythonMapInArrow(func, output, child, isBarrier) =>
```

Comment:
Do we also want to rename logical plans? I see at least `logical.PythonMapInArrow` here, which is inconsistent with the physical plan.